File: practice/castle/castle.java
/*
ID: juwang1
LANG: JAVA
TASK: castle
*/
import java.io.*;
import java.util.*;
import java.util.Collections;


class castle {

  public static void main (String [] args) throws IOException {
    BufferedReader f = new BufferedReader(new FileReader("castle.in"));
    PrintWriter out = new PrintWriter(new BufferedWriter(new FileWriter("castle.out")));
    StringTokenizer st = new StringTokenizer(f.readLine());
    int m = Integer.parseInt(st.nextToken());
    int s = Integer.parseInt(st.nextToken());

    out.println(total+"");
    out.close();
}
}

File: practice/castle/castle.cpp
/*
ID: juwang1
TASK: castle
LANG: C++                 
*/

#include <iostream>
#include <fstream>
#include <string>

using namespace std;

int main() {
    freopen("castle.in", "r", stdin);
  	freopen("castle.out", "w", stdout);
  	ios_base::sync_with_stdio(false);
    int a, b;
    cin >> a; 
    cin >> b;
    int arr[b][a];
    for (int i =0; i<b;i++){
    	for(int j=0;j<a;j++){
    		cin>>arr[i][j];
    	}
    }



    cout << a+b << endl;
    return 0;
}